Tests/iaas/SCS-Spec.MandatoryFlavors.verbose.yaml

```diff
@@ -1,155 +1,199 @@
 reference:
 - field: name
   mandatory_prefix: SCS-
+  optional_prefix: SCSX-
 - field: public
-  default: 'true'
+  default: true
 - field: disabled
-  default: 'false'
-- field: name
+  default: false
 - field: cpus
 - field: ram
 - field: disk
-- field: description
 mandatory:
 - name: SCS-1V-4
   cpus: 1
   ram: 4096
-  disk: 0
-  description: alias=SCS-1V:4
+  scs:cpu-type: shared-core
+  scs:name-v1: SCS-1V:4
+  scs:name-v2: SCS-1V-4
 - name: SCS-2V-8
   cpus: 2
   ram: 8192
-  disk: 0
-  description: alias=SCS-2V:8
+  scs:cpu-type: shared-core
+  scs:name-v1: SCS-2V:8
+  scs:name-v2: SCS-2V-8
 - name: SCS-4V-16
   cpus: 4
   ram: 16384
-  disk: 0
-  description: alias=SCS-4V:16
+  scs:cpu-type: shared-core
+  scs:name-v1: SCS-4V:16
+  scs:name-v2: SCS-4V-16
 - name: SCS-8V-32
   cpus: 8
   ram: 32768
-  disk: 0
-  description: alias=SCS-8V:32
+  scs:cpu-type: shared-core
+  scs:name-v1: SCS-8V:32
+  scs:name-v2: SCS-8V-32
 - name: SCS-1V-2
   cpus: 1
   ram: 2048
-  disk: 0
-  description: alias=SCS-1V:2
+  scs:cpu-type: shared-core
+  scs:name-v1: SCS-1V:2
+  scs:name-v2: SCS-1V-2
 - name: SCS-2V-4
   cpus: 2
   ram: 4096
-  disk: 0
-  description: alias=SCS-2V:4
+  scs:cpu-type: shared-core
+  scs:name-v1: SCS-2V:4
+  scs:name-v2: SCS-2V-4
 - name: SCS-4V-8
   cpus: 4
   ram: 8192
-  disk: 0
-  description: alias=SCS-4V:8
+  scs:cpu-type: shared-core
+  scs:name-v1: SCS-4V:8
+  scs:name-v2: SCS-4V-8
 - name: SCS-8V-16
   cpus: 8
   ram: 16384
-  disk: 0
-  description: alias=SCS-8V:16
+  scs:cpu-type: shared-core
+  scs:name-v1: SCS-8V:16
+  scs:name-v2: SCS-8V-16
 - name: SCS-16V-32
   cpus: 16
   ram: 32768
-  disk: 0
-  description: alias=SCS-16V:32
+  scs:cpu-type: shared-core
+  scs:name-v1: SCS-16V:32
+  scs:name-v2: SCS-16V-32
 - name: SCS-1V-8
   cpus: 1
   ram: 8192
-  disk: 0
-  description: alias=SCS-1V:8
+  scs:cpu-type: shared-core
+  scs:name-v1: SCS-1V:8
+  scs:name-v2: SCS-1V-8
 - name: SCS-2V-16
   cpus: 2
   ram: 16384
-  disk: 0
-  description: alias=SCS-2V:16
+  scs:cpu-type: shared-core
+  scs:name-v1: SCS-2V:16
+  scs:name-v2: SCS-2V-16
 - name: SCS-4V-32
   cpus: 4
   ram: 32768
-  disk: 0
-  description: alias=SCS-4V:32
+  scs:cpu-type: shared-core
+  scs:name-v1: SCS-4V:32
+  scs:name-v2: SCS-4V-32
 - name: SCS-1L-1
   cpus: 1
   ram: 1024
-  disk: 0
-  description: alias=SCS-1L:1
+  scs:cpu-type: crowded-core
+  scs:name-v1: SCS-1L:1
+  scs:name-v2: SCS-1L-1
 - name: SCS-2V-4-20s
   cpus: 2
   ram: 4096
   disk: 20
-  description: alias=SCS-2V:4:20s
+  scs:cpu-type: shared-core
+  scs:disk0-type: ssd
+  scs:name-v1: SCS-2V:4:20s
+  scs:name-v2: SCS-2V-4-20s
 - name: SCS-4V-16-100s
   cpus: 4
   ram: 16384
   disk: 100
-  description: alias=SCS-4V:16:100s
+  scs:cpu-type: shared-core
+  scs:disk0-type: ssd
+  scs:name-v1: SCS-4V:16:100s
+  scs:name-v2: SCS-4V-16-100s
 recommended:
 - name: SCS-1V-4-10
```

Comment:
Wouldn't ``optional_prefix: SCSX-`` imply that we have to use ``SCSX-[...]`` from this line on in the ``recommended`` section of the file? Or does ``optional_prefix: SCSX-`` mean something else?

Reply:
I have no idea what it means, to be quite frank. I only copied it from a file that I saw in the wild. Does it mean anything at all? I mean, it says "optional". I don't seem to find any mention of it in the standard. I think we should just drop that line for good. Give me a minute.

Reply:
I think I copied it from here https://github.com/osism/openstack-flavor-manager/blob/main/flavors.yaml
It's gone now :)